pkg/kube/apis/boshdeploymentcontroller/v1alpha1/types.go

```diff
@@ -8,10 +8,28 @@ import (
 // It's used as input for the Kube code generator
 // Run "make generate" after modifying this file
 
+// Valid values for ref types
+const (
+	ConfigMapType string = "configmap"
+	SecretType    string = "secret"
+	UrlType       string = "url"
```

Comment:
`go vet` is still complaining about some minor things.

```suggestion
	UrlType       string = "url"
	URLType     string = "url"
```